screens/AppNavigator/screens/TransactionsScreen/screens/TransactionsScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react'
+import tailwind from 'tailwind-rn'
+import { translate } from '../../../../../translations'
+import { View, Button, FlatList, TouchableOpacity } from 'react-native'
+import { useEffect, useState } from 'react'
+import { useWalletAPI } from '../../../../../hooks/wallet/WalletAPI'
+import { useWhaleApiClient } from '../../../../../hooks/api/useWhaleApiClient'
+import { Ionicons } from '@expo/vector-icons'
+import { Text, NumberText } from '../../../../../components'
+import { useNavigation } from '@react-navigation/native'
+import { activitiesToViewModel, VMTransaction } from './reducer'
+
+interface TxRow extends VMTransaction {
+  onPress: () => void
```

Comment:
Split this out, VMTransaction also the prop being passed to detail page just without the row press function.